README.md

```diff
@@ -30,25 +30,49 @@ to
 [announce@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/announce).
 See all the [mailing lists](https://www.tensorflow.org/community/forums).
 
-**Tensorflow ROCm port**
-This project is based on TensorFlow 1.14.0. It has been verified to work with the latest ROCm2.6 release.
-Please follow the instructions [here](https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/quick-start.md) to set up your ROCm stack.
-A docker container: **rocm/tensorflow:latest(https://hub.docker.com/r/rocm/tensorflow/)** is readily available to be used:
+## TensorFlow ROCm port
+
+This repository hosts the port of TensorFlow on [ROCm](https://rocm.github.io/)
+open platform. It uses various ROCm technologies such as
+[HIP](https://github.com/ROCm-Developer-Tools/HIP) and
+[MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen).
+For details on TensorFlow ROCm port,
+see [ROCm-specific README file](README.ROCm.md).
+
+- TensorFlow ROCm 1.14 works with ROCm2.6.
+- TensorFlow ROCm 2.0 works with ROCm2.9.
```

Comment:
It sounds like a good idea to document the past ROCm compatible version here. Though we should document TF ROCm 1.15 because it will be the long lasting v1 release.

Will need @sunway513 to decide on the update on this main docs page.